README.md

```diff
@@ -1,5 +1,8 @@
-# units
+# cf_units
 
-A fork of iris units module
+Units of measure.
```

Comment:
How about: `Units of measure as required by the Climate and Forecast (CF) metadata conventions.`?